### ISSUE TITLE
Cannot load characters via Steam Client (Cloud or Local)

### ISSUE THREAD

**Issue by DJLoretz** [MelvorIdle/melvoridle.github.io#965]
Accounts are accessible via the browser with no issue however recently I cannot load any cloud or local saves through the Steam client. I have disabled the Steam Overlay via properties and that did not resolve it. This seemed to have occurred after the April 11th update

1.) Launch the client
2.) There is successful login and cloud connection.
3.) It displays local saves with what I believe are accurate dates/times.
     - Clicking on a local save results in the visual elements shifting slightly left, and characters will not load, after this I cannot 
        replicate the shift until I relaunch the client.
4.) Select Show Cloud Saves
5.) Cloud saves are present and when selected they will do the same shift as with local saves and characters do not load.

**Comment by MelvorIdle** [comment]
Hey!

Are you able to export your Local Save from the character selection screen and import it into the browser version?

If this works - Log into your Cloud account on the browser version and force save that local save so it overwrites the potentially broken Cloud Save. Once that is done, you should be able to load the Cloud Save from the Steam client with no issues.

Let me know how this goes

**Comment by DJLoretz** [comment]
An update with a question: I signed out of the cloud and it seems that I can access one character each time before having to reset the client. It appears I am missing graphical elements as can be seen in the picture, icons and such are missing. I assume this is some sort of save corruption? I validated the files through Steam and it didn't find any missing files, is there a way to reset the account or would I need to remake my account/character to address the issue?

![image](https://user-images.githubusercontent.com/70961984/118379112-3e968480-b58d-11eb-8f06-79f85bce3c1c.png)
